narayana-release-process.sh

```diff
@@ -143,6 +138,33 @@ then
 fi
 cd ..
 
+# Raising PR for quarkus to upgrade Narayana to current version.
+QRKSISSUE = WFLYISSUE
```

Comment:
I'm not sure if it's good to go with the same issue name with Quarkus as it was for WFLY

Reply:
I am using the same issue name because when narayana-release-process.sh is called, WFLYISSUE is coming as third argument, and I was not knowing from where, when and who calls this file. Otherwise I would have used the fourth argument for QRKSISSUE.